frontend/packages/erd-core/src/features/erd/components/ERDRenderer/CommandPalette/CommandPalette.module.css

```diff
@@ -79,6 +79,10 @@
   color: var(--overlay-60);
 }
 
+.escButton {
+  min-width: auto;
```

Comment:
This declaration will override the `min-width` declaration of the `Button` component.

https://github.com/liam-hq/liam/blob/b626663ad42f0911f7a2780f6606b2b01cdb314d/frontend/packages/ui/src/components/Button/Button.module.css#L2